subprojects/ide/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r54/ToolingApiEclipseModelAutoBuildTasksCrossVersionSpec.groovy

```diff
@@ -25,7 +25,7 @@ import org.gradle.tooling.model.eclipse.EclipseProject
 @TargetGradleVersion('>=5.4')
 class ToolingApiEclipseModelAutoBuildTasksCrossVersionSpec extends ToolingApiSpecification {
 
-    @TargetGradleVersion('>=2.6 <5.4')
+    @TargetGradleVersion('>=3.0 <5.4')
```

Comment:
:question: Couldn't it be simplified like this?
```suggestion
    @TargetGradleVersion('<5.4')
```

Reply:
They are not equal because the default `TargetGradleVersion` definition can only be overwritten. Because of that, using the `TargetGradleVersion('<5.4')` annotation will include  Gradle`2.14.1` and earlier versions.